app/assets/javascripts/oxalis/view/right-menu/trees_tab_view.js

```diff
@@ -187,23 +187,23 @@ class TreesTabView extends React.PureComponent<Props, State> {
   getActionsDropdown() {
     return (
       <Menu>
-        <Menu.Item key="shuffleTreeColor">
-          <div onClick={this.shuffleTreeColor} title="Change Tree Color">
+        <Menu.Item key="shuffleTreeColor" onClick={this.shuffleTreeColor}>
+          <div title="Change Tree Color">
```

Comment:
Did you try to move the title to the Menu.Item as well? This way we could get rid of the wrapper divs for this and the next Menu.Items :)